.github/workflows/build.yml

```diff
@@ -1,5 +1,15 @@
 name: "Build and Report Generation"
-on: [push, pull_request]
+on: 
+  push:
+    paths:
+      - '**'
+      - '!docs/**'
```

Comment:
I am not sure if the files under docs/assets/code have any automatic testing process associated with them. i.e., if there is an examination process to make sure that `q1.sql`, when supplied with `q1.env` produce the same output as `q1.output`. I do not see this in the build config, but I will double check to make sure. Otherwise love the idea!

Reply:
Yeah, the code is only there for reference -- not used in any build. The only place that has any reference to that folder is, [Tutorial.md](https://github.com/partiql/partiql-lang-kotlin/blob/11c265e08f45abb7be7ebaa5f51fe58aa4bc0eb3/docs/tutorials/Tutorial.md#partiql-queries-are-sql-compatible) -- just pointing to the larger .env file.